apps/server/src/modules/task/utils/TestHelper.ts

```diff
@@ -1,4 +1,8 @@
+/* istanbul ignore file */
+
+import { ObjectId } from '@mikro-orm/mongodb';
 import { EntityId, TestHelper } from '@shared/domain';
+import { Course } from '@src/entities';
```

Comment:
It is a scope break. Submission and task are bind at the moment. But Course are not part of it and should not add in this helper.

Reply:
really? what else is doing the binding then?